src/include/sof/bit.h

```diff
@@ -8,25 +8,28 @@
 #ifndef __SOF_BIT_H__
 #define __SOF_BIT_H__
 
-/* Zephyr defines this - remove local copy once Zephyr integration complete */
-#ifdef BIT
-#undef BIT
-#endif
+#ifdef __ZEPHYR__
+#include <zephyr/sys/util_macro.h>
+#else
 
 #if ASSEMBLY
 #define BIT(b)			(1 << (b))
 #else
 #define BIT(b)			(1UL << (b))
 #endif
 
-#define MASK(b_hi, b_lo)	\
-	(((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL) << (b_lo))
-#define SET_BIT(b, x)		(((x) & 1) << (b))
 #define SET_BITS(b_hi, b_lo, x)	\
 	(((x) & ((1ULL << ((b_hi) - (b_lo) + 1ULL)) - 1ULL)) << (b_lo))
 #define GET_BIT(b, x) \
 	(((x) & (1ULL << (b))) >> (b))
```

Comment:
not for this PR - I see `GET_BIT()` and `SET_BITS()` defined in Zephyr in DMIC and DAI driver headers with the latter being defined in two headers, maybe they could be moved to a common shared header